ontologies/Examples/MIRO_sample_image_of_artwork_v2.json

```diff
@@ -187,6 +187,7 @@
           "type": "Location",
           "locationType": "iiif-image",
           "url": "https://iiif.wellcomecollection.org/image/V0006724.jpg/info.json",
+          "copyright": "Wellcome "
```

Comment:
This line is missing a comma, which is tripping the CI linting.

Reply:
Thanks I struggled to find the problem file from the error messages.